package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "WATCHPACK_POLLING=true next dev",
```

Comment:
I feel like I would prefer if this wasn't done here. Can you try if using the attached docker-compose.txt file also fixes your issues? Please copy the file contents and paste them into your docker-compose.yml file
[docker-compose.txt](https://github.com/user-attachments/files/16045579/docker-compose.txt)